src/app/gene-plot/gene-plot.component.ts

```diff
@@ -142,14 +143,17 @@ export class GenePlotComponent implements OnChanges {
   }
 
   private get svgHeight(): number {
-    const transcriptsCount = (this.showTranscripts ? this.genePlotModel.gene.transcripts.length : 0) + 1;
+    const transcriptsCount = (
+      this.showTranscripts ?
+        this.genePlotModel.gene.transcripts.length : this.genePlotModel.gene.collapsedTranscripts.length - 1) + 1;
```

Comment:
Maybe there shouldn't be a `-1` in the ternary op here, check how it looks with and without it